api/src/main/java/org/jboss/shrinkwrap/api/Filters.java

```diff
@@ -153,7 +153,7 @@ private static Filter<ArchivePath> createRegExpFilter(String filterClassName, Pa
         StringBuilder classExpression = new StringBuilder();
         for (Package pack : packages) {
             classExpression.append("|");
-            classExpression.append("(.*" + pack.getName().replaceAll("\\.", "\\.") + ".*)");
+            classExpression.append("(.*").append(pack.getName().replaceAll("\\.", ".")).append(".*)");
```

Comment:
Why the change of `+` to `.append`? In other instances in this PR you're doing the opposite.

Wouldn't it be also better to have just a single line of `classExpression.append("|(.*" + pack.getName().replaceAll("\\.", ".")) + ".*)");`?

Reply:
.append() is the recommended way how to work with StringBuilder. It avoids creating unnecessary intermediate String objects. On the other hand, the recommended way to work with String is to use "+".

Reply:
True, missed that. My comment regarding the single line append still stands though

Reply:
That happens and I understand, but it's not a single line since it's inside the for loop.